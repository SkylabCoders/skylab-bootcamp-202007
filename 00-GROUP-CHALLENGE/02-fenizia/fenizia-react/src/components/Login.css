.login-main-container {
  background-color: #fff;
  height: 80vh;
  display: flex;
  flex-direction: column;
  justify-content: center;
}
.login-main-container__wrap {
  margin: auto;
  width: 40vw;
  height: 60vh;
  border: solid 4px #333;
  border-radius: 2%;
  text-align: center;
}
.login-wrap__form__logo {
  width: 20%;
  filter: grayscale(100%);
}
.login-wrap__form {
  display: flex;
  flex-direction: column;
  justify-content: space-evenly;
  height: 100%;
}
.login-wrap__form__btn > button {
  border-style: none;
  padding: 2vh;
  color: black;
  background-color: #ffcc00;
  border: 2px transparent solid;
  width: 10vw;
  min-width: 70px;
}
.login-wrap__form__btn > button:hover {
  border: 2px #333 solid;
}
.login-wrap__form__btn > img {
  width: 40%;
  margin: 2.2vw;
  border: 2px solid transparent;
}
.login-wrap__form__btn > img:hover {
  border: 2px solid #ffcc00;
  border-radius: 9%;
}
.logout-btn {
  background-color: #ffcc00;
  margin-left: 80vw;
  margin-top: 2vh;
  padding: 0.6vw;
  color: #fff;
  border: 2px solid transparent;
}
.logout-btn:hover {
  color: #333;
  border: 2px solid #333;
}
.input__search {
  margin: auto;
}
/* QUERIES */

@media screen and (max-width: 599px) {
  .login-main-container__wrap {
    width: 100vw;
    height: 80vh;
    border: solid 4px transparent;
  }
}
