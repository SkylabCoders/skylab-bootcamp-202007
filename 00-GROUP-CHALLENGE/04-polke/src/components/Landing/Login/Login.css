/*
** FORM INSIDE CONTAINER
*/

.Login__form {
	background-color: var(--form-color-background);
	width: 80%;
	opacity: 0.8;
	display: flex;
	flex-direction: column;
	justify-content: center;
	align-items: center;
	padding-bottom: 10px;
	box-shadow: 2px 2px #888888;
	text-align: center;
}

/*
** FORM ELEMENTS
*/

.Login__form__login-title--colored {
	color: var(--form-color-headers);
	padding-bottom: 5px;
}

.Login__login-with-button {
	width: 100% !important;
}
.Login__login-with-button button {
	width: 80% !important;
	height: 60px;
}
.Login__form__container-input > a {
	color: var(--font-color-input);
}
.Login__form__container-input {
	width: 80% !important;
}

.Login__form__anchor--colored {
	color: var(--form-color-headers);
	font-size: bold;
	font-weight: bolder;
	cursor: pointer;
}

.Login__form__last-button {
	width: 100% !important;
	height: 40px;
	margin-top: 12px;
}

.form__button {
	background-color: var(--bg-color-button);
	border-radius: 5px;
	border: none;
	color: whitesmoke;
	cursor: pointer;
}

.form__button a {
	color: white;
}

.login__log-anonym {
	cursor: pointer;
}

.anchor:hover {
	color: #2d8ff9 !important;
}

/*
** MEDIA QUERIES
*/

@media only screen and (min-width: 600px) {
	.Login__login-with-button button {
		width: 70% !important;
		height: 60px;
	}
	.Login__form__container-input {
		width: 70% !important;
	}
}

@media only screen and (min-width: 1024px) {
	.Login__form {
		min-width: 375px !important;
		max-width: 450px !important;
		display: flex;
		flex-direction: column;
		justify-content: center;
		align-items: center;
		text-align: center;
	}

	.Login__form button {
		width: 300px !important;
	}
	.Login__form__container-input {
		width: 300px !important;
	}
}
