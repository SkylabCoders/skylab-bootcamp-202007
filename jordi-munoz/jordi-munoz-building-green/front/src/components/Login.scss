.login-img:hover {
  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
  border-radius: 50px;
  cursor: pointer;
}
.login-p {
  margin-right: 10px;
}
