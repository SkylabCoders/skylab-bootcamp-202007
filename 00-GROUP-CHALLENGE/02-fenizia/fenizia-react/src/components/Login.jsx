import React, { useState, useEffect } from "react";
import "./Login.css";
import { login, logout, loginWithGoogle } from "../actions/authActions";
import authStore from "../stores/authStore";
import Profile from "./Profile";
import TextInput from "./TextInput";

function Login() {
  const [isLogged, setIsLogged] = useState(authStore.isLogged());
  const [user, setUser] = useState(authStore.getUserProfile());
  const [email, setEmail] = useState("");
  const [password, setPassword] = useState("");

  useEffect(() => {
    authStore.addChangeListener(onAuthChange);

    return () => authStore.removeChangeListener(onAuthChange);
  }, [isLogged, user]);
  function onFieldChange(value, setValue) {
    setValue(value);
  }
  function onAuthChange() {
    setIsLogged(authStore.isLogged());
    setUser(authStore.getUserProfile());
  }

  return (
    <>
      {!isLogged && (
        <>
          <div className="login-main-container">
            <div className="login-main-container__wrap">
              <div className="login-wrap__form">
                <span>
                  <img
                    src="https://image.flaticon.com/icons/svg/1000/1000946.svg"
                    className="login-wrap__form__logo"
                    alt="login"
                  />
                </span>
                <span>Log in</span>

                <div data-validate="Enter username">
                  <TextInput
                    type="text"
                    name="username"
                    placeholder="Username"
                    value={email}
                    onChange={(event) =>
                      onFieldChange(event.target.value, setEmail)
                    }
                  />
                  <span data-placeholder=""></span>
                </div>

                <div data-validate="Enter password">
                  <TextInput
                    type="password"
                    textContentType={"password"}
                    name="pass"
                    placeholder="Password"
                    value={password}
                    onChange={(event) =>
                      onFieldChange(event.target.value, setPassword)
                    }
                  />
                  <span data-placeholder=""></span>
                </div>

                <div className="login-wrap__form__btn">
                  <button onClick={() => login(email, password)}>Login</button>
                  <a href="/register">
                    <p>Sign in</p>
                  </a>
                  <img
                    src="https://icon-library.com/images/login-with-google-icon/login-with-google-icon-3.jpg"
                    onClick={() => loginWithGoogle(email, password)}
                    alt="Log in with google"
                  ></img>
                </div>
              </div>
            </div>
          </div>
        </>
      )}
      {isLogged && (
        <>
          <button className="logout-btn" onClick={() => logout()}>
            Logout
          </button>
          <Profile />
        </>
      )}
    </>
  );
}

export default Login;
