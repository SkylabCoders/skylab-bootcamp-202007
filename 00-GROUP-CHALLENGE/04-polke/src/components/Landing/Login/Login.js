import React, { useState, useEffect } from 'react';
import { FormControl, FormGroup } from 'react-bootstrap';
import {
	login,
	loginGoogle,
	loginAnonymously
} from '../../../actions/loginActions';
import landingStore from '../../../stores/landingStore';
import '../../../shared/generalStyles.css';
import './Login.css';
import PopUpBox from '../PopUpBox/PopUpBox';
import { withRouter } from 'react-router-dom';
import githubApiConst from '../../../shared/githubApiConst';

export default withRouter(function ({ history }) {
	const [isLogged, setIsLogged] = useState(false);
	const [user, setUser] = useState(null);

	const USERDETAIL_PATH = './userDetail';

	useEffect(() => {
		landingStore.addChangeListener(onAuthChange);
		user ? history.push(USERDETAIL_PATH) : history.push('');
		return () => landingStore.removeChangeListener(onAuthChange);
	}, [isLogged, user]);

	function onAuthChange() {
		setIsLogged(landingStore.isLogged());
		setUser(landingStore.getUserProfile());
	}

	function handleLoginClick(event) {
		event.preventDefault();
		const email = document.getElementsByClassName('input-email')[0].value;
		const password = document.getElementsByClassName('input-password')[0].value;
		login(email, password);
	}

	return (
		<>
			<form className="Login__form">
				<h2 className="Login__form__login-title--colored">Log In</h2>
				<FormGroup
					className="Login__login-with-button"
					controlId="username"
					bsSize="large"
				>
					<button
						className="form__button"
						onClick={(event) => {
							event.preventDefault();
							loginGoogle();
						}}
					>
						Google
					</button>
				</FormGroup>
				<FormGroup
					className="Login__login-with-button "
					controlId="password"
					bsSize="large"
				>
					<button className="form__button">
						<a
							href={`https://github.com/login/oauth/authorize?client_id=${githubApiConst.GITHUB_CLIENTID}&scope=${githubApiConst.GITHUB_SCOPE}&login`}
						>
							GitHub
						</a>
					</button>
				</FormGroup>
				<FormGroup
					className="Login__form__container-input"
					controlId="input"
					bsSize="large"
				>
					<FormControl
						className="Login__form__input input-email"
						placeholder="&#128100;"
					/>
					<FormControl
						className="Login__form__input input-password"
						type="password"
						placeholder="&#128272;"
					/>

					<button
						className="Login__form__last-button form__button"
						onClick={(event) => {
							handleLoginClick(event);
						}}
					>
						Enter
					</button>
					<a
						className="Login__form__anchor--colored anchor Login-anchor"
						onClick={(event) => {
							event.preventDefault();
							document.getElementsByClassName('pop-up-modal')[0].style.display =
								'block';
						}}
					>
						Register
					</a>
					{'   |   '}
					<a
						className="login__log-anonym anchor Login-anchor"
						onClick={(event) => {
							event.preventDefault();
							loginAnonymously();
						}}
					>
						Log In Anonymously
					</a>
				</FormGroup>
			</form>
			<PopUpBox />
		</>
	);
});
