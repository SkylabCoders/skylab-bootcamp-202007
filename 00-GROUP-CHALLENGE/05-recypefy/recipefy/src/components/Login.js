import React, { useState, useEffect } from 'react';
import { login, logout, googleLogin } from "../actions/AuthActions";
import authStore from "../stores/AuthStore";
import './Login.css';


function Login(){
   
    const [user, setUser] = useState(authStore.getUserProfile());
    const [email, setEmail] = useState('')
    const [password, setPassword] = useState('');
    // function validateForm(){ 
    //     return email.length > 0 && password.length > 0
    // }
    

    const [isLogged, setIsLogged] = useState(authStore.isLogged())

    useEffect(()=> {
        authStore.addChangeListener(onAuthChange);
       
        return () => authStore.removeChangeListener(onAuthChange);
    }, [isLogged, user]);

    function onAuthChange(){
        setIsLogged(authStore.isLogged());
        setUser(authStore.getUserProfile());
    }

    

    return(
        <div className="login__container">
            <div className="login__box">
                <form>
                    <label for="email">Email</label><br/>
                    <input autoFocus type="email" name="email" className="form__input" value={email} onChange={(e) => { setEmail(e.target.value); console.log(email)}}/><br/>
                    <label>Password</label><br/>
                    <input type="password" className="form__input" value={password} onChange={e => setPassword(e.target.value)}/><br/>
                
                    {!isLogged &&(
                        
                        <button onClick={(event) => { event.preventDefault(); login(email, password)}}>Login</button>
                        )}
                    
                    {isLogged && (
                   
                    <button onClick = {(event) => { event.preventDefault();logout()}}>Logout {user && user.user.email}</button>
                    )}

                </form>
                

                <p>or</p>
                <h2>REGISTER</h2>
                <a onClick={googleLogin}>GOOGLE</a>
                <a>EMAIL</a>
                <a>GUEST</a>
            </div>
        </div>

    );
}

export default Login;