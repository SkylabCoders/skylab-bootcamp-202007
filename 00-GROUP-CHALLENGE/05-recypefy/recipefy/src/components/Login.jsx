import React, { useState, useEffect } from 'react';
import { login, logout, googleLogin, anonymousLogin, createUser } from "../actions/AuthActions";
import authStore from "../stores/AuthStore";
import './Login.scss';


function Login() {
    const [user, setUser] = useState(authStore.getUserProfile());
    const [email, setEmail] = useState('')
    const [password, setPassword] = useState('');
    const [confirmPassword, setConfirmPassword] = useState('')
    const [showRegister, setRegister] = useState(false);
    const [showLogin, setLogForm] = useState(false);
    const [isLogged, setIsLogged] = useState(authStore.isLogged());


    useEffect(() => {
        authStore.addChangeListener(onAuthChange);

        return () => authStore.removeChangeListener(onAuthChange);
    }, [isLogged, user]);

    function onAuthChange() {
        setIsLogged(authStore.isLogged());
        setUser(authStore.getUserProfile());
    }

    function displayLogin() {
        if (!showLogin) {
            setLogForm(true);
        } else {
            setLogForm(false);
        }
    }

    function displayRegister() {
        if (!showRegister) {
            setRegister(true);
        } else {
            setRegister(false);
        }
    }

    function validateForm() {
        return email.length > 0 && password.length > 0
    }


    function validateRegister() {
        if (password === confirmPassword) {
            createUser(email, password);
            alert('You registered successfully!!')
        } else {
            alert('Both passwords has to match.')
        }
    }

    return (
        <div className="login__container">
            <div className="login__box">
                <img className="login__logo--size" src='https://image.flaticon.com/icons/svg/770/770906.svg' />
                <h2 className="login__title">Login with</h2>
                <form>

                    {showLogin && (<>
                        <label className='hide__element' for="email">Email</label><br />
                        <input autoFocus type="email" name="email" className="form__input hide__element" value={email} onChange={(e) => { setEmail(e.target.value) }} /><br />
                        <label className='hide__element' >Password</label><br />
                        <input type="password" className="form__input hide__element" value={password} onChange={e => setPassword(e.target.value)} /><br />


                        {!isLogged && (

                            <button className="login__button hide__element" disabled={!validateForm()} onClick={(event) => { event.preventDefault(); login(email, password) }}>Login</button>
                        )}
                    </>)}

                    {isLogged && (

                        <button className="login__button" onClick={(event) => { event.preventDefault(); logout() }}>Logout {(user && user.user.email) || "Anonymous"}</button>
                    )}

                </form>

                <a onClick={displayLogin}>EMAIL</a>
                <a onClick={googleLogin}>GOOGLE</a>
                <a onClick={anonymousLogin}>ANON</a>
                <h2 className="login__title">OR</h2>
                <a onClick={displayRegister}>REGISTER</a>

                {showRegister && (<>
                    <form className="register__form">

                        <label for="email">Email</label><br />
                        <input autoFocus type="email" name="email" className="form__input" value={email} onChange={(e) => { setEmail(e.target.value) }} /><br />
                        <label>Password</label><br />
                        <input type="password" className="form__input" value={password} onChange={e => setPassword(e.target.value)} /><br />
                        <label>Confirm Password</label><br />
                        <input type="password" className="form__input" value={confirmPassword} onChange={e => setConfirmPassword(e.target.value)} /><br />

                        <button className="login__button" onClick={(event) => { event.preventDefault(); validateRegister() }}>Register</button>
                    </form>
                </>
                )}


            </div>
        </div>

    );
}

export default Login;