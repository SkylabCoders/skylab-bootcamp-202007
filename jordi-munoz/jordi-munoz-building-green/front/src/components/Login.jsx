import React, { useState } from 'react';
import TextField from '@material-ui/core/TextField';
import { makeStyles } from '@material-ui/core/styles';
import Container from '@material-ui/core/Container';
import Button from '@material-ui/core/Button';
import { withRouter } from 'react-router-dom';
import Divider from '@material-ui/core/Divider';
import './Login.scss';



const useStyles = makeStyles((theme) => ({
  root: {
    '& .MuiTextField-root': {
      margin: theme.spacing(1),
      width: '25ch',
    },
  },
}));

export default withRouter(function ({ history }) {
  const classes = useStyles();
  const [email, setEmail] = useState({ text: '' });
  const [pass, setPass] = useState({ text: '' });


  function sendInputs(event, emailParam, passParam) {
    event.preventDefault();
    if (emailParam.text === '' || passParam.text === '') {
      return false;
    } else {
      history.push('/');

    }

    console.log(emailParam, passParam);
  }

  return (
    <form className={classes.root} noValidate autoComplete="off">
      <div style={{
        marginTop: "80px",
        marginBottom: "60px",
        display: "flex",
        flexDirection: "column",
        textAlign: "center"
      }}>
        <Container maxWidth="xs">
          <h2>Accede a tu cuenta</h2>
          <TextField
            value={email.text}
            onChange={event => setEmail({ text: event.target.value })}
            error={email.text === ""}
            id="outlined-email"
            label="e-mail"
            variant="outlined"
          />
          <TextField
            value={pass.text}
            onChange={event => setPass({ text: event.target.value })}
            error={pass.text === ""}
            id="outlined-ubicacion"
            label="Password"
            type="password"
            variant="outlined"
          />
          <div>
            <Button variant="contained" color="primary"
              onClick={(event) => sendInputs(event, email, pass)}
            >
              Acceder
          </Button>
          </div><br />
          <Divider variant="middle" />
          <br />
          <div style={{ display: 'flex', justifyContent: 'center' }}>
            <p className="login-p">Accede con:</p>
            <img className="login-img" src="https://bookassist.org/wp-content/uploads/elementor/thumbs/google_3_520-oc7dqerwmsbfad0t1gveosa6x2uck2bd7y6l2r7txs.jpg" alt="google" />
          </div><br />
          <Divider variant="middle" /><br />
          <Button variant="contained" color="primary"
            href="/user-register"
          >
            Registrarse
      </Button>
        </Container>
      </div>
    </form>
  );
})