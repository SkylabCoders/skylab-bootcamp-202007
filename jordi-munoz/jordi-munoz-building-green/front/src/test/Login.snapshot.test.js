import renderer from 'react-test-renderer';
import React from 'react';
import Login from '../components/Login';
import { BrowserRouter } from 'react-router-dom';

describe('login snapshot', () => {
  const loginTree = renderer.create(
    <BrowserRouter>
      <Login />
    </BrowserRouter>
  );
  it('should match', () => {
    expect(loginTree.toJSON()).toMatchSnapshot();
  });

});