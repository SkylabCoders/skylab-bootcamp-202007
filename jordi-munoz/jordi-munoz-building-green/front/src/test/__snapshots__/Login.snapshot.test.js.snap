// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`login snapshot should match 1`] = `
<form
  autoComplete="off"
  className="makeStyles-root-1"
  noValidate={true}
>
  <div
    style={
      Object {
        "display": "flex",
        "flexDirection": "column",
        "marginBottom": "60px",
        "marginTop": "80px",
        "textAlign": "center",
      }
    }
  >
    <div
      className="MuiContainer-root MuiContainer-maxWidthXs"
    >
      <h2>
        Accede a tu cuenta
      </h2>
      <div
        className="MuiFormControl-root MuiTextField-root"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined Mui-error Mui-error"
          data-shrink={false}
          htmlFor="outlined-email"
          id="outlined-email-label"
        >
          e-mail
        </label>
        <div
          className="MuiInputBase-root MuiOutlinedInput-root Mui-error Mui-error MuiInputBase-formControl"
          onClick={[Function]}
        >
          <input
            aria-invalid={true}
            autoFocus={false}
            className="MuiInputBase-input MuiOutlinedInput-input"
            disabled={false}
            id="outlined-email"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={false}
            type="text"
            value=""
          />
          <fieldset
            aria-hidden={true}
            className="PrivateNotchedOutline-root-2 MuiOutlinedInput-notchedOutline"
          >
            <legend
              className="PrivateNotchedOutline-legendLabelled-4"
            >
              <span>
                e-mail
              </span>
            </legend>
          </fieldset>
        </div>
      </div>
      <div
        className="MuiFormControl-root MuiTextField-root"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined Mui-error Mui-error"
          data-shrink={false}
          htmlFor="outlined-ubicacion"
          id="outlined-ubicacion-label"
        >
          Password
        </label>
        <div
          className="MuiInputBase-root MuiOutlinedInput-root Mui-error Mui-error MuiInputBase-formControl"
          onClick={[Function]}
        >
          <input
            aria-invalid={true}
            autoFocus={false}
            className="MuiInputBase-input MuiOutlinedInput-input"
            disabled={false}
            id="outlined-ubicacion"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={false}
            type="password"
            value=""
          />
          <fieldset
            aria-hidden={true}
            className="PrivateNotchedOutline-root-2 MuiOutlinedInput-notchedOutline"
          >
            <legend
              className="PrivateNotchedOutline-legendLabelled-4"
            >
              <span>
                Password
              </span>
            </legend>
          </fieldset>
        </div>
      </div>
      <div>
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            Acceder
          </span>
        </button>
      </div>
      <br />
      <hr
        className="MuiDivider-root MuiDivider-middle"
      />
      <br />
      <div
        style={
          Object {
            "display": "flex",
            "justifyContent": "center",
          }
        }
      >
        <p
          className="login-p"
        >
          Accede con:
        </p>
        <img
          alt="google"
          className="login-img"
          src="https://bookassist.org/wp-content/uploads/elementor/thumbs/google_3_520-oc7dqerwmsbfad0t1gveosa6x2uck2bd7y6l2r7txs.jpg"
        />
      </div>
      <br />
      <hr
        className="MuiDivider-root MuiDivider-middle"
      />
      <br />
      <a
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
        href="/user-register"
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        >
          Registrarse
        </span>
      </a>
    </div>
  </div>
</form>
`;
